pkg/operators/ebpf/tracer.go

```diff
@@ -95,7 +96,8 @@ func (i *ebpfInstance) populateTracer(t btf.Type, varName string) error {
 	return nil
 }
 
-func (t *Tracer) receiveEvents(gadgetCtx operators.GadgetContext) error {
+func (t *Tracer) receiveEvents(gadgetCtx operators.GadgetContext, wg *sync.WaitGroup) error {
```

Comment:
(Nothing to do with this PR)

The return value of this function doesn't seem to be used anywhere